openid-federation-1_0.xml

```diff
@@ -1303,6 +1311,7 @@
       "exp": 1516298022,
       "metadata": {
         "openid_relying_party": {
+          "client_id": "https://openid.sunet.se",
```

Comment:
This may not be appropriate here either, as you wouldn't include it when using Explicit Registration.